rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/ProductMilestoneEndpoint.java

```diff
@@ -345,6 +347,39 @@ ProductMilestone patchSpecific(
             @Parameter(description = PM_ID) @PathParam("id") String id,
             @Valid @BeanParam PageParameters pageParameters);
 
+    static final String GET_ALL_DELIVERABLE_ANALYZER_OPERATIONS_FILTERED_DESC = "Gets all deliverable analyzer operations executed for this milestone.";
+
+    /**
+     * {@value GET_ALL_DELIVERABLE_ANALYZER_OPERATIONS_FILTERED_DESC}
+     *
+     * @param paginationParameters
+     * @param identifier {@value FILTER_IDENTIFIER_DESC}
+     * @param qualities {@value FILTER_QUALITY_DESC}
+     * @param repoType {@value FILTER_REPOSITORY_TYPE_DESC}
+     * @return
+     */
+    @Operation(
+            summary = GET_ALL_DELIVERABLE_ANALYZER_OPERATIONS_FILTERED_DESC,
+            responses = {
+                    @ApiResponse(
+                            responseCode = SUCCESS_CODE,
+                            description = SUCCESS_DESCRIPTION,
+                            content = @Content(
+                                    schema = @Schema(implementation = DeliverableAnalyzerOperationPage.class))),
+                    @ApiResponse(
+                            responseCode = INVALID_CODE,
+                            description = INVALID_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+                    @ApiResponse(
+                            responseCode = SERVER_ERROR_CODE,
+                            description = SERVER_ERROR_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))) })
+    @GET
+    @Path("/{id}/del-analyzer-operations")
+    Page<DeliverableAnalyzerOperation> getAllDeliverableAnalyzerOperations(
```

Comment:
Why two endpoints this one and `OperationEndpoint.getAllDeliverableAnalyzerOperation()` ?

Reply:
I started the implementation of the public endpoint OperationEndpoint (which can be used for all the operations, in future we could have more than just DeliverableAnalyzerOperations) to get all the deliverable analyzer operations. When I had to query the del analyzer operations specific for a ProductMilestone, I started struggling with the path to use as I had too many nested paths (e.g. /operations/del-analyzer-operations/milestone/{id}), and also not in line with the new "standards" after the 2.0 refactoring. So I opted to use the ProductMilestoneEndpoint and so something similar to the already existing `/product-milestones/{id}/delivered-artifacts`. The future operations might not be connected to a ProductMilestone, so using only the ProductMilestoneEndpoint did not seem correct to me.

Reply:
Ok I can also implement endpoints for the "normal" operations, but since we have none at the moment, I thought I could wait for them. Would you like them now?